cli.sh

```diff
@@ -2538,12 +2539,13 @@ git hooks version
         return
     fi
 
-    CURRENT_VERSION=$(grep -E "^# Version: .*" "$0" | head -1 | sed -E "s/^# Version: //")
-
+    CURRENT_VERSION=$(execute_git "$GITHOOKS_CLONE_DIR" rev-parse --short HEAD)
```

Comment:
Would it be difficult to recreate the version number we had before?
```
VERSION_DATE=$(date +%y%m.%d%H%M)
VERSION_COMMIT=$(git rev-parse --short=6 HEAD)
```

Reply:
But what would that mean :-) we always get a new version number?
I think it should be git based on tags (semantic versioning)... or then simply the changeset number?

Reply:
Sorry, should have been clearer. I thought we could ask `git log` to output the date of the current HEAD commit.

Reply:
I'll leave it up to you, if this is easy, it might be nice.
Otherwise just resolve the conflict please then I'll merge this PR. 👌